vhdbuilder/packer/install-dependencies.sh

```diff
@@ -479,12 +475,16 @@ K8S_VERSIONS="
 1.18.6-hotfix.20200723.1
 1.18.8-hotfix.20200924
 1.18.10
+1.18.10-hotfix.20210118
 1.18.14
+1.18.14-hotfix.20210118
 1.19.0
 1.19.1-hotfix.20200923
 1.19.3
 1.19.6
-1.20.0
+1.19.6-hotfix.20210118
+1.19.7-hotfix.20210122
```

Comment:
Not sure if `1.19.7-hotfix.20210122` will be available or we'll have another hotfix name for 1.19.7.

Reply:
Please ignore the comment, the artifacts should be available. the release pipeline errors are due to the pipeline trying to override them